LibTrixi.jl/src/api_jl.jl

```diff
@@ -77,8 +77,8 @@ end
 
 
 function trixi_nelements_global_jl(simstate)
-    _, _, solver, cache = mesh_equations_solver_cache(simstate.semi)
-    return nelementsglobal(solver, cache)
+    mesh, _, solver, cache = mesh_equations_solver_cache(simstate.semi)
+    return nelementsglobal(mesh, solver, cache)
```

Comment:
This is required when using Trixi.jl v0.7.16 or above.

Reply:
Yes, I guess you need to bump the minimum version then 👍